src/components/document-list/_macro.njk

```diff
@@ -11,7 +11,7 @@
                 {% if document.thumbnail %}
                     <div
                         class="ons-document-list__item-image{{ ' ons-document-list__item-image--file' if document.metadata.file }}"
-                        aria-hidden="true"
+                        aria-label="{{ document.title.text }}"
```

Comment:
We need to move this to the `<a>` rather than the `<div>`, as `aria-label`s are only valid on focusable elements not static ones like `<div>`s so this would be ignored by SRs.

It's not ideal having two links with the same label here but maybe this is the most pragmatic solution if we need to keep this link in.

Can we also remove the `tabindex="-1"` on the image link, so users can access it via the keyboard?